### DISCUSSION TITLE
Upstream Server : context cancelled

### DISCUSSION THREAD

**Discussion by jeevangt** [tchiotludo/akhq#1249] [Q&A]
Hello All when I am trying to hit AKHQ URL and searching for topic I am getting error 
Error proxying to upstream server :  context cancelled. 

After this I am getting 504 error on AKHQ UI. 

Need help. Not able to know how to fix it.

**Comment by tchiotludo** [comment]
Maybe you can share something useful? (Logs, version, configuration files, ...) Whatever that could help?

**Reply by jeevangt** [reply]
Version I am using 0.21.0
Now I am getting error when I am hitting GET point api/auths with my host url
LoginEnabled : false, formenabled  :false.

**Comment by jeevangt** [comment]
I am implemented same config in another cluster and error is same. 

Also ot is throwing 504 error while connecting to proxy server. 

Actually what happen when akhq call to event hub it need to pass proxy server. At the time of proxy it is throwing 504 error. 

So which config need to add not able to understand. 

I  am using JWT token and outh2 proxy.

**Reply by tchiotludo** [reply]
event hub = azure event hub ?

**Reply by jeevangt** [reply]
Event hub is the aws cluster.

**Comment by jeevangt** [comment]
Hey can you tell me how I can copy my trustore and keystore file in value.yaml.

I able to see my job creating folder trustore.p12 file at the path mention in value.yaml and not the actual file. 

So I want from my workspace trusstore value can be pass into value.yaml
